uk-UA_3.x/administrator/language/uk-UA/uk-UA.com_installer.ini

```diff
@@ -18,8 +7,9 @@ COM_INSTALLER_CONFIRM_UNINSTALL="Ви дійсно бажаєте видалит
 COM_INSTALLER_CURRENT_VERSION="Встановлено"
 COM_INSTALLER_DISCOVER_FILTER_SEARCH_DESC="Пошук у назві виявлених розширень. Префікс з ID: пошук розширення по ID."
 COM_INSTALLER_DISCOVER_FILTER_SEARCH_LABEL="Пошук за виявленими розширеннями"
-COM_INSTALLER_ENABLED_UPDATES_1=", 1 недоступний сайт увімкнутий"
```

Comment:
Це неправильний переклад. Потрібно приблизно так:

`1 відключений раніше сервер оновлень успішно включено.`

Коментар з файлу для цієї функції:

```
	/**
	 * Enables any disabled rows in #__update_sites table
	 *
	 * @return  boolean result of operation
	 *
	 * @since   1.6
	 */
```